svelteUi/src/routes/create/+page.svelte

```diff
@@ -0,0 +1,102 @@
+<script lang="ts">
+	let fields = { firstName: '', lastName: '', email: '', phone: '', dateOfBirth: '' };
+
+	//Handles POST request
+	const handleSubmit = async () => {
+		try {
```

Comment:
show errors from api with its corresponding fields or at the end of the form if no field provided